cmd/crictl/sandbox.go

```diff
@@ -30,6 +31,20 @@ import (
 	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
+type sandboxBySort []*pb.PodSandbox
+
+func (a sandboxBySort) Len() int      { return len(a) }
```

Comment:
We should also sort sandbox by created time.

Reply:
Only compare CreatAt?

Reply:
Make sense. I think you are right.